File: Basic/Operator/219158G.c
#include <stdio.h>
#define intll long long

void solution() {

	int n;
	scanf("%d", &n);

	printf("%lld\n", 1ll * n * (n + 1) / 2);

}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Operator/219158G.go
package main

import (
	"fmt"
)

func solution() {

	var n int
	fmt.Scanf("%d", &n)

	fmt.Printf("%d\n", n*(n+1)/2)
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
